x-pack/plugins/uptime/server/lib/requests/get_last_successful_check.ts

```diff
@@ -33,6 +32,18 @@ export const getLastSuccessfulStepParams = ({
     ],
     query: {
       bool: {
+        must_not: [
+          ...REMOVE_NON_SUMMARY_BROWSER_CHECKS.must_not,
```

Comment:
i don't this complex filter is necessary here, just add a filter for summary document and that should be enough.